modules/nw-rfhe-subscribing-redfish-events.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+:_content-type: PROCEDURE
+[id="nw-rfhe-troubleshooting_{context}"]
+= Subscribing to Redfish Events
+
+To subscribe to Redfish Events, create a BMCEventSubscription CRD.
+
+[source,yaml]
+----
+    apiVersion: metal3.io/v1alpha1
+    kind: BMCEventSubscription
+    metadata:
+      name: worker-1-events
+    spec:
+       hostName: ostest-worker-1 <1>
+       destination: https://events.apps.corp.example.com/webhook <2>
+       context: “SomeUserContext”
+       httpHeadersRef:
+         name: some-secret-name
+         namespace: some-namespace
+    status:
+      errorMessage: ""
+      subscriptionID: aa618a32-9335-42bc-a04b-20ddeed13ade
+----
+
+<1> `hostName`: Specifies the name or uuid of the worker node on which Redfish events are generated.
+
+<2> `destination`: Specifies the Baremetal Hardware Event proxy service, for example `https://<hw-event-proxy_service_name>/webhook`. The `hw-event-proxy` service name can be obtained by issuing the following command:
+
+[source,yaml]
+----
+oc -n cloud-native-events  get route hw-event-proxy -o jsonpath='{.spec.host}'
```

Comment:
What is this command doing? Some context to the command would be nice. Also, it would be good to provide an example of what the hardware event you get returned looks like.

Reply:
Now under scope of https://github.com/openshift/openshift-docs/pull/41832